bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -329,6 +329,13 @@ spec:
                 - patch
                 - update
                 - watch
+            - apiGroups:
+              - packages.operators.coreos.com
```

Comment:
Hi @qpdpQ 
Could you please run `make generate-all` to align the indents for the permissions? Thank you!